src/components/views/rooms/RoomHeader.tsx

```diff
@@ -57,6 +86,41 @@ export default function RoomHeader({ room }: { room: Room }): JSX.Element {
                     </BodyText>
                 )}
             </div>
+
+            {!isDirectMessage && (
+                <BodyText
+                    as="div"
+                    size="sm"
+                    weight="medium"
+                    className="mx_RoomHeader_members"
+                    aria-label={_t("%(count)s members", { count: memberCount })}
+                    onClick={(e: React.MouseEvent) => {
+                        togglePanel(RightPanelPhases.RoomMemberList);
+                        e.stopPropagation();
+                    }}
+                >
+                    <AvatarStack>
+                        {members.slice(0, 3).map((member) => {
```

Comment:
Might be worth copying the algorithm from our existing Facepile component which prioritises which users to show (users you have DMs with, users which have avatars etc)

Reply:
Looks like we should switch to `FacePile`. Didn't know about this component. will update.